spellcheck.pl

```diff
@@ -60,6 +60,9 @@ sub spellcheck_check_word
     }
 
     return if $word =~ m{^/}; # looks like a path
+    if (Irssi::settings_get_bool('expand_escapes')) {
+        $word =~ s/\\(['"\\])/$1/g;
```

Comment:
I don't think that's right.
My Irssi (1.0.7) doesn't expand `\'` to `'` even when `expand_escapes` is enabled.
I've filed separate bug (https://github.com/jwilk/irssi-spellcheck/issues/2) for proper support for `expand_escapes`.

Reply:
I see, we're both right. This was due to the fix that landed in Irssi 1.1.0 https://github.com/irssi/irssi/pull/709 but I think it won't hurt on earlier versions

Reply:
I dropped this patch as it doesn't make sense and the Irssi feature has a bug